app/models/newsletter_signup.rb

```diff
@@ -0,0 +1,5 @@
+class NewsletterSignup < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true
+end
```

Comment:
Do I even need a model if we're not saving in the database? It seemed like we would need one for validations but I'm not sure I got that right.